google/resource_google_project.go

```diff
@@ -155,7 +172,12 @@ func resourceGoogleProjectRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("labels", p.Labels)
 
 	if p.Parent != nil {
-		d.Set("org_id", p.Parent.Id)
+		switch p.Parent.Type {
+		case "organization":
+			d.Set("org_id", p.Parent.Id)
```

Comment:
add `d.Set("folder_id", "")` here. and the same for the `org_id field in the folder case below. The reason is that if you move a project from a folder to an org outside of Terraform, we want Terraform to clear the other field when reading the state.

Reply:
Done